core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -107,17 +109,7 @@ trait RawlsApiService
     with VersionApiService
     with ServicePerimeterApiService {
 
-  val multiCloudWorkspaceServiceConstructor: RawlsRequestContext => MultiCloudWorkspaceService
```

Comment:
These are not needed here - they're all inherited from the included traits.
If they were to be included here, they should have an override qualifier.